test/framework/kuma-net/udp/server.go

```diff
@@ -0,0 +1,120 @@
+package udp
```

Comment:
I don't think we want to use "kuma-net" anymore, right? so rename this directory, for example.

Reply:
renamed it to `test/framework/network`